src/gst-plugins-ugly.mk

```diff
@@ -25,4 +25,8 @@ define $(PKG)_BUILD
         --docdir='$(1)/sink' \
         --with-html-dir='$(1)/sink'
     $(MAKE) -C '$(1)' -j '$(JOBS)' install
+
+    # some .dlls are installed to lib - no obvious way to change
+    $(and $(BUILD_SHARED),
```

Comment:
>Is there a rationale of using it rather than if?

I don't know. These changes was introduced by @tonytheodore in https://github.com/mxe/mxe/pull/1157